drivers/include/net/netdev2.h

```diff
@@ -74,6 +74,9 @@ typedef enum {
     NETDEV2_EVENT_RX_COMPLETE,      /**< finished receiving a packet */
     NETDEV2_EVENT_TX_STARTED,       /**< started to transfer a packet */
     NETDEV2_EVENT_TX_COMPLETE,      /**< finished transferring packet */
+#ifdef MODULE_OPENTHREAD
+    NETDEV2_EVENT_TX_COMPLETE_DATA_PENDING, /**< finished transferring packet and has data pending flag **/
```

Comment:
Hmm, probably because at that time I didn't see a use case for a separate `TX_COMPLETE_DATA_PENDING` and since I started examining the trac status, this seemed like a sensible default. I guess the driver didn't support the feature that would cause such a status.

But there was no other particular reason I can remember.